types/audit-details.d.ts

```diff
@@ -172,6 +173,34 @@ declare global {
         value: string;
       }
 
+      export interface LightWallet {
```

Comment:
My thinking here was to have the Details.LightWallet format mirror the budgets format as closely as possible bc that’s probably not going to change much but UX could.

Keeping the info for the diagnostics section separate from budget results made the audit/renderer logic much cleaner, even if the info is a bit duplicative in some cases.